### DISCUSSION TITLE
How to backup MinIO users with secret-key

### DISCUSSION THREAD

**Discussion by Sandhyaranitp** [minio/minio#15212] [Q&A] [answered]
Team, Can you please let me know the ways to backup Minio console users with credential. I tried below query but not able to get the "secret-key" associated with each user.
mc admin user list <alias> --json
mc admin user info <alias> <user> --json

Also is it possible to pull all details with single query?

**Comment by vadmeste** [comment, marked as answer]
@Sandhyaranitp there is a new feature called IAM export/import, you need to upgrade to the latest version and use that.

**Comment by harshavardhana** [comment]
Use `mc admin cluster iam export/import`

**Reply by Sandhyaranitp** [reply]
@vadmeste @harshavardhana 
I see the config.json is getting created post executing this command, but I don’t see any users created inside minIO console.
Also most of the entries are like below
"accessKey": "YOUR-ACCESS-KEY-HERE",
"secretKey": "YOUR-SECRET-KEY-HERE"

will this "mc admin cluster iam export <alias>" pull even the console users?

**Reply by harshavardhana** [reply]
It pulls all users @Sandhyaranitp

**Reply by Sandhyaranitp** [reply]
@harshavardhana... I run below command to pull the details but I get some error when I execute export. Can you please here 

$mc alias set minio https://minio-****@***.com <user> <pwd>                  --I have passed console url and credentials
mc: Configuration written to `/..../config.json`. Please update your access credentials.
Added `minio` successfully.

$mc admin cluster iam export minio   
mc: <ERROR> Unable to export IAM info. Failed to parse server response (unexpected end of JSON input):.

**Reply by harshavardhana** [reply]
You need to use latest MinIO @Sandhyaranitp you haven't upgraded your server.

**Comment by syedgufran95** [comment]
The easiest way to backup is to copy .minio.sys folder. This folder contains all configurations and iam users. just copy the folder and past it inside the new instance. The folder location is where all your buckets resides on the system

**Comment by harshavardhana** [comment]
@syedgufran95 no that is wrong correct way is `mc admin cluster iam export`
